man/fragments/ex-navs_tab.Rmd

```diff
@@ -0,0 +1,223 @@
+  )
+  b$screenshot(filename = file.path(
+    rprojroot::find_package_root_file(),
+    "man/figures/", file_out
+  ))
+
+  invisible(file_out)
+}
+
+include_screenshot <- function(x, options) {
+  file_name <- paste0(options$label, ".png")
+  alt <- options$fig.alt
+  if (is.null(alt)) {
+    alt <- sprintf("Screenshot of a %s() example.", options$label)
+  }
+  img <- sprintf('![](%s "%s")', file_name, alt)
+  knitr::asis_output(img)
+}
+
+knitr::opts_chunk$set(
+  render = function(x, options) {
+    if (isTRUE(SCREENSHOT)) {
+      render_screenshot(x, options)
+    }
+    include_screenshot(x, options)
+  }
+)
+```
+
+## A basic example
+
+This first example creates a simple tabbed navigation container with two tabs.
+The tab name and the content of each tab are specified in the `nav()` calls
+and `navs_tab()` creates the tabbed navigation around these two tabs.
+
+```{r navs_tab-basic, fig.alt = "Screenshot of a basic navs_tab() example."}
+library(htmltools)
+
+navs_tab(
+  nav(title = "One", p("First tab content.")),
+  nav(title = "Two", p("Second tab content."))
+)
+```
+
+In the rest of the examples, we'll include links among the tabs (or pills) in the navigation controls.
+
+```{r echo = TRUE, eval = TRUE, results="hide"}
+link_shiny <- tags$a(shiny::icon("github"), "Shiny", href = "https://github.com/rstudio/shiny", target = "_blank")
+link_posit <- tags$a(shiny::icon("r-project"), "Posit", href = "https://posit.co", target = "_blank")
+```
+
+## `navs_tab()`
+
+You can fully customize the controls in the navigation component.
+In this example, we've added a direct link to the Shiny repository using `nav_item()`.
+We've also included a dropdown menu using `nav_menu()`
+containing an option to select a third tab panel
+and another direct link to Posit's website.
+Finally, we've separated the primary tabs on the left
+from the direct link and dropdown menu on the right
+using `nav_spacer()`.
+
+```{r navs_tab}
+navs_tab(
+  nav(title = "One", p("First tab content.")),
+  nav(title = "Two", p("Second tab content.")),
+  nav_spacer(),
+  nav_item(link_shiny),
+  nav_menu(
+    title = "Other links",
+    align = "right",
+    nav("Three", p("Third tab content")),
+    nav_item(link_posit)
+  )
+)
+```
+
+## `navs_pill()`
+
+`navs_pill()` creats a navigation container that behaves exactly like `navs_tab()`,
+but the tab toggles are _pills_ or button-shaped.
+
+```{r navs_pill}
+navs_pill(
+  nav(title = "One", p("First tab content.")),
+  nav(title = "Two", p("Second tab content.")),
+  nav_spacer(),
+  nav_item(link_shiny),
+  nav_menu(
+    title = "Other links",
+    align = "right",
+    nav("Three", p("Third tab content")),
+    nav_item(link_posit)
+  )
+)
+```
+
+## `navs_tab_card()`
+
+The tabbed navigation container can also be used in a `card()` component
+thanks to `navs_tab_card()`.
+Learn more about this approach in the
+[article about Cards](https://pkgs.rstudio.com/bslib/articles/cards.html).
```

Comment:
Maybe worth mentioning `sidebar` argument (and link to sidebar article)?

Reply:
@cpsievert Do you know the ID of the heading that will talk about sidebars in tabbed cards?

Reply:
it looks like `#multi-page-layout` would be a good option.